cli/default-reporter/test/index.ts

```diff
@@ -1055,7 +1062,7 @@ test('logLevel=default', async () => {
   const output = await firstValueFrom(output$.pipe(skip(2), take(1)))
   expect(output).toBe(`Info message
 ${formatWarn('Some issue')}
-${formatError('ERROR', 'some error')}`)
+${formatError('ERR_PNPM_SOME_CODE', 'some error')}`)
```

Comment:
A bunch of tests were actually failing, but fortunately they were very easy to update. Note how this test throws `SOME_CODE` above, but we weren't checking for it correctly when calling `expect(...)`.

https://github.com/pnpm/pnpm/blob/55dc29428e24eae16ccd291c6a8011f5bafd79af/cli/default-reporter/test/index.ts#L346